index.html

```diff
@@ -5705,6 +5717,63 @@ <h3>Complex Language Markup</h3>
 
     </section>
 
+    <section class="appendix normative">
+      <h2>Verification Algorithm</h2>
+
+      <p>
+This section contains the algorithm that <a>conforming verifier processors</a>
+run when verifying a <a>verifiable credential</a> or a <a>verifiable
+presentation</a>. Any algorithm can be used that produces output that
+is equivalent to the algorithm defined in this section. The algorithm takes a
+byte stream (<var>inputBytes</var>) and media type (<var>inputMediaType</var>)
+as input and produces an object (<var>verified</var>) that contains a
+verification status (<var>verified</var>.<var>status</var>),
+verified document (<var>verified</var>.<var>document</var>),
+media type (<var>verified</var>.<var>mediaType</var>),
+and optional warning  (<var>verified</var>.<var>warnings</var>) and
+error (<var>verified</var>.<var>errors</var>) information.
+      </p>
+
+      <ol class="algorithm">
+        <li>
+Set <var>verified</var> to an empty object.
+        </li>
+        <li>
+Verify that the securing mechanism in use on <var>inputBytes</var> has not been
+tampered with, using <var>inputMediaType</var> as additional input to the
+algorithm. See Section <a href="#securing-verifiable-credentials"></a>
+for details.
```

Comment:
Yes, we can call out specific algorithms, like this (generalized) one:

https://w3c.github.io/vc-data-integrity/#verify-proof

or this (specific) one:

https://w3c.github.io/vc-di-eddsa/#verify-proof-eddsa-rdfc-2022

What I'm trying to avoid is limit future cryptosuites from being used by being too specific here. Your suggestion to define an interface that various securing specs implement sounds like a good direction. I'll add that to this PR.

Reply:
Yep. A couple things I notice when comparing this call to those definitions are that:
* this call doesn't provide the [proof options](https://www.w3.org/TR/vc-data-integrity/#dfn-proof-options) that https://www.w3.org/TR/vc-data-integrity/#verify-proof expects,
* https://w3c.github.io/vc-di-eddsa/#verify-proof-eddsa-rdfc-2022 doesn't pass them through,
* neither of those expects the media type that this call does pass, and 
* https://www.w3.org/TR/vc-data-integrity/#verify-proof expects its input to have been already parsed to something that contains object fields, but this algorithm doesn't do that.

So there may be two dispatch layers here, possibly expressed as registries: first based on the media type to parse, and then based on the cryptosuite to verify. But I'm sure that omits some important details, and I'll look forward to seeing what you write.